RELEASE_NOTES.rst

```diff
@@ -1,3 +1,64 @@
+coala-bears 0.11.0
+==================
+
+This release has been brought to you by 30 contributor contributing over 101
+commits.
+
+**Bug Fixes**
+
+- ``InvalidLinkBear`` now extracts links from markdown correctly.
+  [Issue 1338](https://github.com/coala/coala-bears/issues/1338)
+- ``InvalidLinkBear`` now extracts all the links in a line. Earlier it used to
+  extract only the first match.
+- ``PyCodeStyleBear`` now uses user settings along with default settings.
+  Earlier it used to ignore default settings if user changed one of them.
+- ``PyImpostSortBear`` now recongnizes known third party imports correctly.
+  [Issue 1510](https://github.com/coala/coala-bears/issues/1510)
+
+**New Bears**
+
+	**CSS**
+	- ``StyleLintBear`` - Lints css files using stylelint
+
+	**Elm**
+	- ``ElmLintBear`` - Formats source code according to given set of rules.
+
+	**Generic**
+	- ``LicenseCheckBear`` - Checks given files for license searching start or
+      end of files for lines belonging to various licenses.
+	- ``DuplicateFileBear`` - Checks for duplicate files.
+
+**Enhancements**
+- ``FilenameBear`` can now recognize kebabcase naming convention.
+- ``FilenameBear`` can predict naming conventions for wide-spread languages like
+  java, javascript and python. Set the ``file_naming_convention`` to ``auto`` to
+  use this feature.
+- ``GitCommitBear`` can now enforce issue reference in commits.
+- ``GitCommitBear`` now recognizes ``user/repo#12`` type issue references.
+- ``JSONFormatBear`` notifies if run on empty files.
+- ``JSONFormatBear`` tells the lines and cols at which the error occured instead
```

Comment:
s/cols/columns (generally please don't abbreviate in relnotes, especially non-standard ones)